bc/assets/templates/includes/header.html

```diff
@@ -140,10 +140,47 @@
               <div class="w-28 text-sm font-medium mr-1">
                 {% include './action-button.html' with link="https://free.law/donate/" text="Donate" size='sm' color='saffron' %}
               </div>
-              <div class="w-20 text-sm font-medium sm:text-md {% if user.is_authenticated %}hidden{% endif %}">
-                {% url 'sign-in' as sign_in %}
-                {% include './action-button.html' with link=sign_in text="Sign In" size='sm' %}
-              </div>
+              {% if user.is_authenticated %}
```

Comment:
Correct me if I'm wrong here, but won't this cause chaos due to the CDN? How do we address that?

Reply:
Oh, right. My bad. Let's hope you're right! :)